scripts/ci/run-tests

```diff
@@ -16,5 +17,10 @@ def run(command):
     return check_call(command, shell=True)
 
 
+_, args = OptionParser(usage="%prog TEST_DIR ...").parse_args()
```

Comment:
To make this simpler, I would probably just pass ``sys.argv`` through to the underlying command instead of trying to parse arguments here, defaulting to just ``unit/ functional/`` if ``sys.argv`` is empty.